docs/source/install.rst

```diff
@@ -61,87 +69,16 @@ Windows
-
-    python setup.py install
-
-Python 3.5
-~~~~~~~~~~
-Download and install the most recent version of
-`Visual Studio Community Edition <https://www.visualstudio.com/vs/community/>`__
-and then install using
-
-    python setup.py install
-
-
-32-bit or other versions of Python
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-You can build 32-bit version of the code on windows using mingw32.
-
-First, get and install `mingw32 <http://www.mingw.org/>`__. Then, you'll need to edit distutils.cfg. This is usually found somewhere like C:\Python27\Lib\distutils\distutils.cfg. Add these lines::
-
-    [build]
-    compiler=mingw32
-
-Then in the statsmodels directory do::
-
-    python setup.py build
-    python setup.py install
-
-OR
-
-You can build 32-bit Microsoft SDK. Detailed instructions can be found on the
-Cython wiki `here <https://github.com/cython/cython/wiki/CythonExtensionsOnWindows>`__.
-The gist of these instructions follow. You will need to download the free
-Windows SDK C/C++ compiler from Microsoft. You must use
-the **Microsoft Windows SDK for Windows 7 and .NET Framework 3.5 SP1** to be
-comptible with Python 2.7, 3.1, and 3.2. The link for the 3.5 SP1 version is
-
-`https://www.microsoft.com/en-us/download/details.aspx?id=18950 <https://www.microsoft.com/en-us/download/details.aspx?id=18950>`__
-
-For Python 3.3 and 3.4, you need to use the **Microsoft Windows SDK for Windows 7 and .NET Framework 4**,
-available from
-
-`https://www.microsoft.com/en-us/download/details.aspx?id=8279 <https://www.microsoft.com/en-us/download/details.aspx?id=8279>`__
-
-For 7.0, get the ISO file GRMSDKX_EN_DVD.iso for AMD64. After you install this,
-open the SDK Command Shell (Start -> All Programs ->
-Microsoft Windows SDK v7.0 -> CMD Shell). CD to the statsmodels directory and type::
-
-    set DISTUTILS_USE_SDK=1
-
-To build a 64-bit application type::
-
-    setenv /x64 /release
-
-To build a 32-bit application type::
-
-    setenv /x86 /release
-
-The prompt should change colors to green. Then proceed as usual to install::
-
-    python setup.py build
-    python setup.py install
-
-For 7.1, the instructions are exactly the same, except you use the download
-link provided above and make sure you are using SDK 7.1.
-
-If you want to accomplish the same without opening up the SDK CMD SHELL, then
-you can use these commands at the CMD Prompt or in a batch file.::
-
-    setlocal EnableDelayedExpansion
-    CALL "C:\Program Files\Microsoft SDKs\Windows\v7.0\Bin\SetEnv.cmd" /x64 /release
-    set DISTUTILS_USE_SDK=1
-
-Replace `/x64` with `/x86` and `v7.0` with `v7.1` as needed.
+Getting the right compiler is especially confusing for Windows users. Over time,
+Python has been built using a variety of different Windows C compilers.
+`This guide <https://wiki.python.org/moin/WindowsCompilers>`_ should help
+clarify which version of Python uses which compiler by default.
 
+Alternatively, you can use the
+`MinGW-w64 toolchoin <http://mingw-w64.org/doku.php>`__.
```

Comment:
This is not supported anymore for python > 3.4

Reply:
I'll just remove this. There's tons of support now for the "official" C compiler for Windows nowadays, we should aim people towards the most common and supported tools.